code/game/gamemodes/cult/cult_items.dm

```diff
@@ -72,6 +72,171 @@
+			playsound(src, pick('sound/weapons/bulletflyby.ogg', 'sound/weapons/bulletflyby2.ogg', 'sound/weapons/bulletflyby3.ogg'), 75, 1)
+			return 1
+		else
+			playsound(src, 'sound/weapons/parry.ogg', 75, 1)
+			owner.visible_message("<span class='danger'>[owner] parries [attack_text] with [src]!</span>")
+			return 1
+	return 0
+
+/obj/item/twohanded/required/cult_bastard/afterattack(atom/target, mob/user, proximity, click_parameters)
+	. = ..()
+	if(dash_toggled && charges)
+		dash(user, target)
+		return
+	if(!proximity)
+		return
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = target
+		if(H.stat != CONSCIOUS)
+			var/obj/item/device/soulstone/SS = new /obj/item/device/soulstone(src)
+			if(SS.attack(H, user))
+				shards += SS
+			else
+				qdel(SS)
+	if(istype(target, /obj/structure/constructshell) && shards.len)
+		var/obj/item/device/soulstone/SS = shards[1]
+		if(SS.transfer_soul("CONSTRUCT",target,user))
+			shards -= SS
+
+/obj/item/twohanded/required/cult_bastard/proc/charge()
+	charges = Clamp(charges + 1, 0, 1)
+	playsound(src, 'sound/magic/exit_blood.ogg', 25, 1)
+	if(istype(loc, /mob/living))
+		to_chat(loc, "<span class='cultitalic'>[src] is ready for another blood jaunt.</span>")
+
+/obj/item/twohanded/required/cult_bastard/proc/dash(mob/user, atom/target)
+	if(!charges)
+		return
+	if(isliving(target))
+		return
+	if(target.density)
+		return
+	var/turf/T = get_turf(target)
+	if(target in view(user.client.view, get_turf(user)))
+		var/obj/spot1 = new /obj/effect/temp_visual/dir_setting/cult/phase/out(get_turf(user), user.dir)
+		user.forceMove(T)
+		playsound(T, 'sound/magic/enter_blood.ogg', 25, 1)
+		var/obj/spot2 = new /obj/effect/temp_visual/dir_setting/cult/phase(get_turf(user), user.dir)
+		spot1.Beam(spot2,"sendbeam",time=20)
+		charges = Clamp(charges - 1, 0, 1)
+		addtimer(CALLBACK(src, .proc/charge), charge_rate)
+
+
+/datum/action/innate/spin2win
+	name = "Geometer's Fury"
+	desc = "You draw on the power of the sword's ancient runes, spinning it wildly around you as you become immune to most attacks."
+	background_icon_state = "bg_demon"
+	button_icon_state = "sintouch"
+	var/cooldown = 0
+	var/mob/living/carbon/human/holder
+	var/obj/item/twohanded/required/cult_bastard/sword
+
+/datum/action/innate/spin2win/Grant(mob/user, obj/bastard)
+	. = ..()
+	sword = bastard
+	holder = user
+
+/datum/action/innate/spin2win/IsAvailable()
+	if(iscultist(holder) && cooldown <= world.time)
+		return TRUE
+	else
+		return FALSE
+
+/datum/action/innate/spin2win/Activate()
+	cooldown = world.time + 250
+	holder.changeNext_move(50)
+	holder.apply_status_effect(/datum/status_effect/sword_spin)
+	sword.spinning = TRUE
+	sword.block_chance = 100
+	addtimer(CALLBACK(src, .proc/stop_spinning), 50)
+	holder.update_action_buttons_icon()
+
+/datum/action/innate/spin2win/proc/stop_spinning()
+	sword.spinning = FALSE
+	sword.block_chance = 50
+	sword.slowdown -= 1
```

Comment:
?

Reply:
Oh I cut the slowdown forgot to remove the reversal slowdown, it's pointless since the var does nothing for held items.

Reply:
@Robustin not true, if the item has the ```HAND_SLOW``` (or w/e) flag, then ```slowdown``` DOES work.